### ISSUE TITLE
Some of islands in my server does not have 100x100 island range

### ISSUE THREAD

**Issue by Klymenos** [tastybento/ASkyBlock-Bugs-N-Features#903]
Thank you for filing a bug report. Please complete these sections to help speed resolution.

**Server version? Remember to say if it is Spigot/Bukkit/Paper etc.**

**Plugin version:**

**Is this a new install or upgrade from earlier plugin version? If upgrade, what version did you use before?**

**(Optional) What other plugins are you using now? (do /plugins and paste text from the console log here)**

**Description of issue. What happened?**

**Steps to make this happen**

1. 
2.
3.

**What do you think should happen?**

**Comment by Klymenos** [comment]
My server version is 1.12 and spigot

I use AcidIsland V3.0.9.4

I got same issues before versions.
Some island has 100x100 island range it is ok but some island does not have 100x100.
I checked up island info it said this island have 100 island range but i counted blocks from the island center.The A rim is far away 50 blocks from the center, it is same for the B rim and the C rim. There is a problem with the D rim. The distance of D rim from the center is 9.To sum up the distance of the rims from the center is 50 blocks in some island but i can not tell same things for the others.
